tests/test_plexosdb_sqlite.py

```diff
@@ -12,8 +14,13 @@ def db_empty() -> "PlexosSQLite":
 
 
 @pytest.fixture
-def db(data_folder) -> PlexosSQLite:
-    return PlexosSQLite(xml_fname=data_folder.joinpath(DB_FILENAME))
+def db(data_folder) -> Generator[PlexosSQLite, None, None]:
+    xml_fname = data_folder.joinpath(DB_FILENAME)
+    xml_copy = data_folder.joinpath(f"copy_{DB_FILENAME}")
+    shutil.copy(xml_fname, xml_copy)
+    db = PlexosSQLite(xml_fname=xml_copy)
+    yield db
+    xml_copy.unlink()
```

Comment:
can we create the copy on a `tmp_path` instead of the data folder?

Reply:
oh yeah, that makes sense. I was trying to use with TemporaryFile, but it was not deleting the file when the tests completed, I can try the directory approach

Reply:
Done